text/0249-v2-experiments.md

```diff
@@ -199,6 +199,82 @@ Disadvantages:
   mitigate this downside by keeping the old unstable package around, but that
   leads to duplicated classes.
 
+### Option 5: superset/subset releases
+
+Instead of *splitting* the stable and experimental APIs into two different
+packages, we release two builds of the same source code, where the *experimental*
+build is a superset of the *stable* build.
+
+This can be done automatically, by using jsii to strip out experimental APIs
+from the **public interface** of the stable release. The implementation
+(`.js` files) will be the same in both releases, so stable APIs in
+`aws-cdk-lib` can safely use experimental APIs internally (though not in
+their public API surfaces).
+
+**How does stripping APIs from the public work?**
+
+| TypeScript            | JavaScript | Java, Python, C#, Go            |
+|-----------------------|------------|---------------------------------|
+| Stripped from `.d.ts` | N/A        | No bindings generated by `jsii` |
```

Comment:
Minor: I feel like this table should be organized that each language is a row, not a column... kind of difficult to read right now.